<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowTelemetryViewerOptions</class>
 <widget class="QDialog" name="WindowTelemetryViewerOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telemetry viewer options</string>
  </property>
  <widget class="QPushButton" name="btn_ShowChart">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Chart</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_LoadedTelemetry">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded telemetry:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CurrentTelemetryFilename">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_LoadTelemetry">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Telemetry</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_AddVariable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Add variable to chart:</string>
   </property>
   <widget class="QComboBox" name="cb_AddVariable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddVariable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="lb_VariablesOnChart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_AddedVars">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variables on chart:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_Lap">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Laps to show: (, delim)</string>
   </property>
   <widget class="QCheckBox" name="cb_Laps_AllLaps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All laps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Laps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can specify certain lap with a comma (,) delimited list. Lap 0 is the first lap.</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_XAxis">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Distance to show:</string>
   </property>
   <widget class="QLineEdit" name="le_DistanceStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_DistanceEnd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DistanceStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DistanceEnd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>End:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DistanceMax">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DistanceZoomIn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DistanceZoomOut">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DistanceLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_DistanceRight">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ClearVariables">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
